django/thunderstore/repository/migrations/0053_schedule_chunked_package_caching.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 3.1.7 on 2024-05-21 12:10
+
+import pytz
+from django.db import migrations
+
+TASK = "thunderstore.repository.tasks.update_chunked_package_caches"
+
+
+def forwards(apps, schema_editor):
+    CrontabSchedule = apps.get_model("django_celery_beat", "CrontabSchedule")
+    PeriodicTask = apps.get_model("django_celery_beat", "PeriodicTask")
+
+    schedule, _ = CrontabSchedule.objects.get_or_create(
+        minute="0",
+        hour="*",
+        day_of_week="*",
+        day_of_month="*",
+        month_of_year="*",
+        timezone=pytz.timezone("UTC"),
+    )
+
+    PeriodicTask.objects.get_or_create(
+        crontab=schedule,
+        name="Update APIV1ChunkedPackageCache",
+        task=TASK,
+    )
```

Comment:
Can you find a way to add this argument to the task programmatically within this migration already?

![image](https://github.com/thunderstore-io/Thunderstore/assets/8225825/5ce4c00b-c7b3-44ae-8ed2-7789bbae599b)

Otherwise they'll pile up in the queue if it lasts longer than the schedule

Reply:
If you do find a way, don't worry about recreating the migration history in this PR, just inline edit it in here (I assume you would, explicitly noting it anyway just in case)

Reply:
@MythicManiac adding it was trivial, but I wasn't sure if the 300 in the pic was just a reference. I used it but let me know if a different value would make more sense.